framework/commerce/types/product.ts

```
@@ -43,6 +43,7 @@ export type Product = {
   variants: ProductVariant[]
   price: ProductPrice
   options: ProductOption[]
+  metafields: { edges: any[] }
```

My first TypeScript :joy: 

It actually worked really well as I could still develop while ignoring the errors